Custom-Audio-Driver/Custom-Audio-Driver/DefaultAudioDevice.swift

```diff
@@ -293,7 +296,7 @@ extension DefaultAudioDevice: OTAudioDevice {
         return recording
     }
     func estimatedRenderDelay() -> UInt16 {
-        return UInt16(playoutDelay)
+        return UInt16(min(self.playoutDelay, UInt32(DefaultAudioDevice.kMaxPlayoutDelay)))
     }
     func estimatedCaptureDelay() -> UInt16 {
         return UInt16(recordingDelay)
```

Comment:
no UInt16 casting needed here??

Reply:
we need it because it is an uint32

Reply:
hmm looks like I may have commented on old code.. UInt16  cast ok now
why not add a range here too?

Reply:
I completely agree it should be validated, because the same issue we were facing with playoutDelay can also occur. I created https://jira.vonage.com/browse/OPENTOK-49302 for tech debt day